corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -119,6 +123,21 @@ def process_changes(self, since, forever):
                             self._record_checkpoint_in_datadog()
                         self._record_change_success_in_datadog(change)
                     self._record_change_in_datadog(change, timer)
+
+                    if timer.duration > self.slow_threshold:
+                        extra = change.metadata.to_json() if change.metadata else {}
+                        extra.update({
+                            'duration': '{} seconds'.format(timer.duration),
+                            'threshold': self.slow_threshold,
+                        })
+                        client.captureMessage(
+                            u'{} change slow to process'.format(self.pillow_id),
+                            extra=extra,
+                            time_spent=int(timer.duration * 1000),  # Convert to milliseconds
+                            fingerprint=[self.pillow_id],
```

Comment:
that makes sense. my reasoning is that domains have extremely differently structured data so I think it would be helpful when something comes up to see the same domain causing issues. I think it should be in the change.metadata so it'll be under extra. Is there a way in sentry to see that in a nice way?